### ISSUE TITLE
Doxygen hrefs contain extra padding

### ISSUE THREAD

**Issue by hyoklee** [HDFGroup/hdf5#1931]
See the following section in [1]

```
Don't like what you see? - You can help to improve this Reference Manual
Complete the survey linked near the top of this page!
We treat documentation like code: Fork the [HDF5 repo](https://github.com/HDFGroup/hdf5), make changes, and create a [pull request](https://github.com/HDFGroup/hdf5/pulls) !
See the [Reference Manual (RM) Page Template](https://docs.hdfgroup.org/hdf5/develop/_r_m_t.html) for general guidance.
```
Extra spaces are added  before ```,make changes``` and ```!``` when link points to GitHub.com.

[1] https://docs.hdfgroup.org/hdf5/develop/_r_m.html

**Comment by derobins** [comment]
This is because padding-right in hdf5doxy.css is set to 16 for hrefs. This is easy to fix, but I don't know what else will look mis-formatted if we change this.

**Comment by derobins** [comment]
Fixed in #1934
